_includes/shared/who-we-are.html

```diff
@@ -4,9 +4,12 @@
         <h4>Who we are</h4>
 
         <p>
-            We are Adobe Consulting Services.
-            We build solutions on top of Adobe products including Adobe Experience Manager (AEM, fka CQ, fka Communique).
-            <a href="http://www.adobe.com/consulting/" target="_blank">Learn more.</a>
+            We are the AEM developer community.
+            We are developers that like AEM, and like building solutions on top of Adobe products including Adobe Experience Manager (AEM, fka CQ, fka Communique).        
+        </p>
+
+        <p>
+            Please note that the name of this Github organization and repository are from a legacy ownership of the this project. Adobe Consulting Services is no longer maintains this project, instead the AEM developer community does.
```

Comment:
`Adobe Consulting Services is no longer maintains this project, instead the AEM developer community does.` odd phrasing here.